tests/input_files/homogenized-constrained-mixture-membrane-cyl-aniso-expl-nofibdir.dat

```diff
@@ -28,6 +28,7 @@ LINEAR_SOLVER                   1
 THETA                           0.5
 --------------------------------------------------------------------SOLVER 1
 SOLVER                          UMFPACK
+IFPACK_XML_FILE                 xml/preconditioner/ifpack.xml
```

Comment:
This must not be necessary! We need to be able to construct direct solvers without having to specify a preconditioner.